sputnikdao-factory2/src/lib.rs

```diff
@@ -342,6 +371,42 @@ impl SputnikDAOFactory {
         }
     }
 
+    pub fn store_factory_metadata(
```

Comment:
what is the point in metadata if upgrade factory is called directly without storing contract code first?

Reply:
The point is to just have a on-chain record referencing code_hash, commit id, and changelog url.  If this is accomplished via the DAO storing the blob and then subsequently making the UpgradeRemote proposal then I can remove.  I just thought it would be better to have referenced in the factory itself.

As a side note, even store_contract_metadata technically doesn't do anything as this data can be encapsulated even within the proposal description itself.  I can remove for now and revisit in a separate standards doc on factory upgrades.

Reply:
Created #196 to review store_contract_metadata

Reply:
Resolved above issues by removing store_factory_metadata for now will discuss in detail on #196